js/mediator/mediator.js

```diff
@@ -0,0 +1,67 @@
+/** @format */
+
+class Mediator {
+  constructor(element) {
+    this.element = element;
+  }
+
+  refs = {
+    buttons: document.getElementsByClassName("movie-info__list__btn"),
+    contentContainer: document.querySelector("#movie_content"),
+    movieList: document.getElementsByClassName("movie-info__list"),
+    movieListRight: document.getElementsByClassName("movie-info__list-right"),
+  };
+
+  addMovieList(event) {
+    const { textContent } = event.target;
+
+    for (let i = this.refs.movieList.length - 1; i >= 0; --i) {
+      this.refs.movieList[i].remove();
+      this.refs.movieListRight[i].remove();
+    }
+    apiService.getInfo({
+      query: textContent,
+      firstCallBack: murkupBtnMovie,
+      value: "person",
+    });
+  }
+
+  addMovieContent(event) {
+    const { textContent } = event.target;
+
+    this.addCurrentClass(event);
+    this.refs.contentContainer.innerHTML = "";
+    apiService.getInfo({
+      query: textContent,
+      firstCallBack: murkupContentSection,
+      value: "movie",
+    });
+  }
+
+  createArrBtn(element) {
+    return Array.prototype.slice.call(element);
+  }
+
+  addCurrentClass(event) {
+    const buttons = this.createArrBtn(this.refs.buttons);
```

Comment:
You can just use spread operator to get an array of buttons
```suggestion
    const buttons = [...this.refs.buttons];
```